sylvia/src/types.rs

```diff
@@ -235,6 +235,36 @@ pub struct ReadyExecutorBuilderState;
 ///
 /// # fn main() {}
 /// ```
+///
+/// It is also possible to call execution methods on other contract's interface:
+///
+/// ```rust
+/// pub mod interface {
+///     # use cosmwasm_std::{Response, StdResult, StdError};
+///     # use sylvia::interface;
+///     # use sylvia::types::ExecCtx;
+///     
+///     #[interface]
+///     pub trait Interface {
+///         type Error: From<StdError>;
+///
+///         #[sv::msg(exec)]
+///         fn exec_method(&self, ctx: ExecCtx) -> StdResult<Response>;
+///     }
+/// }
+///
```

Comment:
nit. Those wrapping newlines will be doubled in the docs and look weird. You can check it with `cargo doc`.

Reply:
Done